src/components/mgt-people-picker/mgt-people-picker.scss

```diff
@@ -22,7 +22,9 @@ $input-border-top: var(--input-border-top, $input-border);
 
 $input-hover-color: var(--input-hover-color, #323130);
 $input-focus-color: var(--input-focus-color, #0078d4);
-$input-background-color: var(--input-background-color, var(--people-list-background-color, white));
+$input-background-color: var(--input-background-color, white);
+
+$people-list-background-color: var(--people-list-background-color, #f1f1f1);
```

Comment:
You should be able to reuse `--input-background-color` here and avoid this CSS prop, no? 

`--people-list-background-color` was replaced by `--input-background-color` few versions ago. If we need a new css prop, I recommend using a more generic name and keeping the `--people-list-background-color` as is for compatibility.

Reply:
The color for the selected items was hard coded with a transparent number. But yeah I was curious why people-list-background-color was in there but never used anywhere else. I can name it differently. No prob.

Reply:
Also what about $person-item-background-color... I can't think of a better name

Reply:
Got it. How about `--selected-person-background-color`?